src/herder/test/UpgradesTests.cpp

```diff
@@ -2691,23 +2691,66 @@ TEST_CASE("upgrades serialization roundtrip", "[upgrades]")
         initUpgrades.setConfigUpgrade(ltx, configUpgradeSet->getKey());
         ltx.commit();
     }
+    {
```

Comment:
We don't need sections here because state doesn't need to be reset. Add sections here would just make the tests take longer.

Reply:
I like sections just for the sake of documentation and I think the runtime difference is marginal enough to not worry about it much, but up to you.

Reply:
We've made decisions in the past to not add unnecessary sections because our test cases can take a while to run. This test is small enough that I think it's fine to not have sections.